src/layout/Surface.js

```diff
@@ -85,7 +85,7 @@
         surfaceButtons.enter().append("button").classed("surface-button",true)
             .each(function (button, idx) {
                 var el = context._surfaceButtons[idx] = d3.select(this)
-                    .attr("class", "surface-button " + button.class)
+                    .classed(button.class,button.class !== undefined)
```

Comment:
I don't know for sure and suspect its probably ok - but I suspect it is passed to the DOMNode which is probably wasteful at best.